nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -265,6 +265,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The Wordpress module got support for installing language packs through `services.wordpress.sites.<site>.languages`.
 
+- The `mullvad-vpn` module is now available cross-platform, instead of being limited to only x86_64.
```

Comment:
Thanks for handling this!

However, I think the more important thing to call out is that the default package changed -- this means that users who relied on the GUI will need to add the `mullvad-vpn` package to their system (or change the package back) in order to access the GUI.

Maybe something like: "The default package for `services.mullvad-vpn.package` was changed to `pkgs.mullvad`, allowing cross-platform usage of Mullvad. `pkgs.mullvad` only contains the Mullvad CLI tool, so users who rely on the Mullvad GUI will want to change it back to `pkgs.mullvad-vpn`, or add `pkgs.mullvadvpn` to their environment."

Reply:
Oops, good call out. I lost some context here since it's been a while since I made the change, but yes that's what we need to warn people about.

I think your suggestion works fine and fits with the rest of the release notes as well. I've updated to use that text.